clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangelogConsumerImpl.java

```diff
@@ -152,9 +165,39 @@ public VeniceChangelogConsumerImpl(
     if (changelogClientConfig.getViewName() == null || changelogClientConfig.getViewName().isEmpty()
         || changelogClientConfig.isBeforeImageView()) {
       this.storeName = changelogClientConfig.getStoreName();
+      rocksDBStorageEngineFactory = null;
+      // The in memory storage engine only relies on the name of store and nothing else. We use an unversioned store
+      // name
+      // here in order to reduce confusion (as this storage engine can be used across version topics).
+      InMemoryStorageEngine inMemoryStorageEngine = new InMemoryStorageEngine(storeName);
+      this.chunkAssembler = new InMemoryChunkAssembler(inMemoryStorageEngine);
     } else {
       this.storeName =
           VeniceView.getViewStoreName(changelogClientConfig.getStoreName(), changelogClientConfig.getViewName());
+      Properties rocksDBBufferProperties = new Properties();
+      String rocksDBBufferPath = changelogClientConfig.getRocksDBChunkAssemblerStoragePath();
+      if (rocksDBBufferPath == null || rocksDBBufferPath.isEmpty()) {
+        throw new VeniceException(
+            "rocksDBChunkAssemblerStoragePath must be configured for consuming view store: " + storeName);
+      }
+      rocksDBBufferProperties.put(DATA_BASE_PATH, rocksDBBufferPath);
```

Comment:
Shall we create a new folder in the path specified by the application?
I worried about that users might specify an non-empty folder, which contains other important data and the CDC consumer can drop all the files in that folder.